template-sample-app-2/template/cypress/integration/tileset_page_spec.js

```diff
@@ -33,6 +33,8 @@ describe('Tileset Page', () => {
     zoomOut.click();
 
     // Total aggregated sum
-    getAggregationSumWidget().findByText('411M').should('exist');
+    getAggregationSumWidget()
+      .findByText(/^4\d\d(\.\d+)M$/)
```

Comment:
It's not pretty, but it's more robust… (at least while the number of buildings in the dataset is < 500 M)

Reply:
Ok, I'm gonna try to deal with it in another PR here: https://github.com/CartoDB/carto-react-template/pull/336, where I'm facing same issue

Reply:
Ok, I wasn't able to finish the CI upgrade on #336, so I moved the WIP to #337 (where locally all e2e tests pass for v3)